src/libraries/skinDynLib/test/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+# Copyright: (C) 2017 Fondazione Istituto Italiano di Tecnologia
+# Authors: Silvio Traversaro
+# CopyPolicy: Released under the terms of the GNU GPL v2.0.
+
+add_executable(skinContactTest skinContactTest.cpp)
+target_link_libraries(skinContactTest skinDynLib)
+add_test(NAME skinContactTest COMMAND skinContactTest)
```

Comment:
Fixed. If you see it often, I think this is due to the lack of proper tooling to insert the final new line/check the missing new line in files. Any suggestions?